content/en/agent/faq/log4j_mitigation.md

```diff
@@ -50,7 +54,7 @@ Instructions are different depending on the Linux distribution:
 
 **Note**: Use `start` and `stop`, because `restart` does not pick up the service configuration change.
 
-**Note**: The `/etc/init/datadog-agent.conf` file is overwritten when the Agent is re-installed, upgraded, or downgraded, you must run these steps again if you upgrade, downgrade, or reinstall the Agent until you upgrade the Agent to v7.32.2/v6.32.2 or above.
+**Note**: The `/etc/init/datadog-agent.conf` file is overwritten when the Agent is re-installed, upgraded, or downgraded, you must run these steps again if you upgrade, downgrade, or reinstall the Agent until you upgrade the Agent to v7.32.3/v6.32.3 or above.
```

Comment:
this can actually still mention `7.32.2`/`6.32.2` (since applying this mitigation is redundant with 7.32.2/6.32.2), but continuing to mention `7.32.2`/`6.32.2` might be more confusing than helpful.

```suggestion
**Note**: The `/etc/init/datadog-agent.conf` file is overwritten when the Agent is re-installed, upgraded, or downgraded, you must run these steps again if you upgrade, downgrade, or reinstall the Agent until you upgrade the Agent to v7.32.2/v6.32.2 or above.
```

Reply:
I changed it to 7.32.3 so that everything indicates that the best result is to be on 7.32.3 or later.

Reply:
makes sense